projects/libs/core-library-typescript/test/consensus/block-builder.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { types } from "../../src/common-library/types";
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import * as sinonChai from "sinon-chai";
+import { stubInterface } from "ts-sinon";
+import BlockBuilder from "../../src/consensus/block-builder";
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+const transactionPool = stubInterface<types.TransactionPoolClient>();
+const virtualMachine = stubInterface<types.VirtualMachineClient>();
+
+function randomUInt() {
+  return Math.floor(Math.random() * Number.MAX_SAFE_INTEGER);
+}
+
+function aRandomTransaction(): types.Transaction {
+    return {
+        version: 1,
```

Comment:
We are currently using `version = 0` for transactions (and appendices) and have a test in `TransactionHandler` (`if (transaction.version !== 0) {`) and another place - if this test passes, does it mean that we don't verify transaction structure at this endpoint? Otherwise, any test using `version = 1` should've failed because of it.

Reply:
👍